README.md

```
@@ -0,0 +1,137 @@
+make install
+```
+
+## Quick Start
+
+1. Set your API key:
+
+```bash
+export ANTHROPIC_API_KEY="sk-..."
+```
+
+2. Run the shell:
```

markdownlint-cli2 (0.20.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)